modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/MiniCart.js

```diff
@@ -23,27 +23,52 @@ import {
 	CHANGE_ORDER,
 } from '../../utilities/eventsDefinitions';
 import {showErrorNotification} from '../../utilities/notifications';
-import CartItemsList from './CartItemsList';
 import MiniCartContext from './MiniCartContext';
-import Opener from './Opener';
-import Wrapper from './Wrapper';
-import {regenerateOrderDetailURL, resolveView} from './util/index';
+import {
+	ADD_PRODUCT,
+	CART,
+	HEADER,
+	ITEM,
+	ITEMS_LIST,
+	ITEMS_LIST_ACTIONS,
+	OPENER,
+	ORDER_BUTTON,
+	ORDER_IS_EMPTY,
+	REMOVE_ALL_ITEMS,
+	REVIEW_ORDER,
+	SUBMIT_ORDER,
+	SUMMARY,
+	VIEW_DETAILS,
+	YOUR_ORDER,
+} from './util/constants';
+import {
+	normalizePartialObject,
+	regenerateOrderDetailURL,
+	summaryDataMapper,
+} from './util/index';
+import {DEFAULT_LABELS} from './util/labels';
+import {DEFAULT_VIEWS, resolveCartViews} from './util/views';
 
 function MiniCart({
 	cartActionURLs,
-	cartItemsListView,
-	cartView,
+	cartViews,
 	displayDiscountLevels,
+	displayTotalItemsQuantity,
+	itemsQuantity,
+	labels,
+	onAddToCart,
 	orderId,
 	spritemap,
+	summaryDataMapper,
+	toggleable,
 }) {
-	const AJAX = ServiceProvider.DeliveryCartAPI('v1');
+	const CartResource = ServiceProvider.DeliveryCartAPI('v1');
 
-	const [isOpen, setIsOpen] = useState(false),
+	const [isOpen, setIsOpen] = useState(!toggleable || false),
```

Comment:
This `|| false` doesn't make sense.

Whatever `toggleable` is, `!toggleable` is going to be a boolean, so you either have:

- `true || false` (ie. `true`); or:
- `false || false` (ie. `false`).

So, just make it `!toggleable`.

Reply:
Leftover... :p Thanks!